plugins/woocommerce/client/legacy/js/admin/woocommerce_admin.js

```diff
@@ -1,93 +1,136 @@
-			$('.wc-progress-form-content').block({
-				message: null,
-				overlayCSS: {
-					background: '#fff',
-					opacity: 0.6
-				}
-			});
-			return true;
-		} );
+		$( '.woocommerce-progress-form-wrapper .button-next' ).on(
+			'click',
+			function () {
+				$( '.wc-progress-form-content' ).block( {
+					message: null,
+					overlayCSS: {
+						background: '#fff',
+						opacity: 0.6,
+					},
+				} );
+				return true;
+			}
+		);
 
 		// Field validation error tips
 		$( document.body )
-
-			.on( 'wc_add_error_tip', function( e, element, error_type ) {
+			.on( 'wc_add_error_tip', function ( e, element, error_type ) {
 				var offset = element.position();
 
 				if ( element.parent().find( '.wc_error_tip' ).length === 0 ) {
-					element.after( '<div class="wc_error_tip ' + error_type + '">' + woocommerce_admin[error_type] + '</div>' );
-					element.parent().find( '.wc_error_tip' )
-						.css( 'left', offset.left + element.width() - ( element.width() / 2 ) - ( $( '.wc_error_tip' ).width() / 2 ) )
+					element.after(
+						'<div class="wc_error_tip ' +
+							error_type +
+							'">' +
+							woocommerce_admin[ error_type ] +
+							'</div>'
+					);
+					element
+						.parent()
+						.find( '.wc_error_tip' )
+						.css(
+							'left',
+							offset.left +
+								element.width() -
+								element.width() / 2 -
+								$( '.wc_error_tip' ).width() / 2
+						)
 						.css( 'top', offset.top + element.height() )
 						.fadeIn( '100' );
 				}
-			})
-
-			.on( 'wc_remove_error_tip', function( e, element, error_type ) {
-				element.parent().find( '.wc_error_tip.' + error_type ).fadeOut( '100', function() { $( this ).remove(); } );
-			})
-
-			.on( 'click', function() {
-				$( '.wc_error_tip' ).fadeOut( '100', function() { $( this ).remove(); } );
-			})
+			} )
+
+			.on( 'wc_remove_error_tip', function ( e, element, error_type ) {
+				element
+					.parent()
+					.find( '.wc_error_tip.' + error_type )
+					.fadeOut( '100', function () {
+						$( this ).remove();
+					} );
+			} )
+
+			.on( 'click', function () {
+				$( '.wc_error_tip' ).fadeOut( '100', function () {
+					$( this ).remove();
+				} );
+			} )
 
-			.on( 'blur', '.wc_input_decimal[type=text], .wc_input_price[type=text], .wc_input_country_iso[type=text]', function() {
-				$( '.wc_error_tip' ).fadeOut( '100', function() { $( this ).remove(); } );
-			})
+			.on(
+				'blur',
+				'.wc_input_decimal[type=text], .wc_input_price[type=text], .wc_input_country_iso[type=text]',
+				function () {
+					$( '.wc_error_tip' ).fadeOut( '100', function () {
+						$( this ).remove();
+					} );
+				}
+			)
 
 			.on(
 				'change',
-				'.wc_input_price[type=text], .wc_input_decimal[type=text], .wc-order-totals #refund_amount[type=text]',
-				function() {
-					var regex, decimalRegex,
+				'.wc_input_price[type=text], .wc_input_decimal[type=text], .wc-order-totals #refund_amount[type=text], ' +
```

Comment:
Everything related to the class `wc_input_variations_price` in this file has been added by this PR.

Reply:
> Is this part suppose to handle the number parsing?
Nope, it has the same behavior that the variations fields have. The numbers aren't formatted, we were only checking the thousands and decimal pointing.

Reply:
I'm not sure if it will worth adding that validation to the old product management experience, but if you think that we need it I'll be happy to add it in a follow-up PR.